airflow/providers/ssh/operators/ssh.py

```diff
@@ -89,7 +90,8 @@ def execute(self, context) -> Union[bytes, str, bool]:
                     self.log.info(
                         "ssh_hook is not provided or invalid. " "Trying ssh_conn_id to create SSHHook."
                     )
-                    self.ssh_hook = SSHHook(ssh_conn_id=self.ssh_conn_id, timeout=self.timeout)
+                    self.ssh_hook = BaseHook.get_connection(self.ssh_conn_id)
```

Comment:
Is the `ssh_hook` proper name? I think this will be a connection now?

Reply:
I have withdrawn this change. I will deal with it in a separate PR.